python/cuda_cooperative/cuda/cooperative/experimental/block/_block_reduce.py

```diff
@@ -89,41 +125,46 @@ def reduce(dtype, threads_in_block, binary_op, methods=None):
     )
 
 
-def sum(dtype, threads_in_block):
-    """Computes a block-wide reduction for thread\ :sub:`0` using addition (+) as the reduction operator.
-    Each thread contributes one input element.
+def sum(dtype, threads_in_block, items_per_thread=1, methods=None):
```

Comment:
Is `methods=` mean to be public-facing kwarg? (it's not documented).

- If so, we should document it
- If not, let's either hide it behind a `**kwargs` or add a leading `_` to indicate it's for internal use

Reply:
Yes, I added some cursory documentation for it. We need to describe the protocol better.